server_admin/topics/authentication/flows.adoc

```diff
@@ -7,18 +7,17 @@ To view all the flows, actions, and checks, each flow requires:
 
 .Procedure
 . Click *Authentication* in the menu.
```

Comment:
A single-step procedure looks weird.  Please move it down to just before line 15, "Click on the _Browser_